sql/parser/sql.y

```diff
@@ -1202,13 +1202,22 @@ table_like_option:
 | ALL {}
 
 index_def:
-  opt_unique INDEX opt_name '(' name_list ')' opt_storing
+  INDEX opt_name '(' name_list ')' opt_storing
   {
     $$ = &IndexTableDef{
-      Name:    Name($3),
-      Unique:  $1,
-      Columns: NameList($5),
-      Storing: $7,
+      Name:    Name($2),
+      Columns: NameList($4),
+      Storing: $6,
+    }
+  }
+| UNIQUE INDEX opt_name '(' name_list ')' opt_storing
```

Comment:
Because the functionality is already supported elsewhere in the grammar? Or something else?